### PR TITLE
Validator dockerize

### PR DESCRIPTION
**Pull request by kriii**
Dockerfile renewal for the validator.

### CODE REVIEW

**Comment by kriii** [comment]
I checked the status of the node but `lastBlockNumber` is `-1`, and nothing changes.I'll reopen this after addressing it.

**Comment by platfowner** [comment]
@kriii , is it ready to resume the PR review? Please let us know.

**Comment by kriii** [comment]
Sorry, I made another minor change, so it is ready for now.

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,6 @@
-FROM node:10.14
+FROM node:16.14
 WORKDIR /app
```
Comment (liayoo) [inline Dockerfile:2]: minor) can we keep the directory name (ain-blockchain)?

File: README.md [hunk 2]
```diff
@@ -157,13 +156,16 @@ docker build -t ain-blockchain .
 ```
 - Pull Docker image
 ```
-docker pull ainblockchain/blockchain-database
+docker pull ainblockchain/ain-blockchain
 ```
 - Run with Docker image
 ```
-docker run -e ACCOUNT_INJECTION_OPTION=keystore KEYSTORE_FILE_PATH=/path/to/keystore --network="host" -d ainblockchain/ain-blockchain:latest
+docker run -e ACCOUNT_INJECTION_OPTION=private_key -e PRIVATE_KEY=your_private_key --network="host" -d ainblockchain/ain-blockchain:latest
```
Comment (liayoo) [inline README.md:163]: 🙌

File: README.md [hunk 3]
```diff
@@ -157,13 +156,16 @@ docker build -t ain-blockchain .
...
+docker run -e ACCOUNT_INJECTION_OPTION=keystore -e KEYSTORE_FILE_PATH=/path/to/keystore" --network="host" -d ainblockchain/ain-blockchain:latest
```
Comment (liayoo) [inline README.md:164]: `"`  is missing after the KEYSTORE_FILE_PATH=

File: README.md (hunk 2 above)
Comment (liayoo) [reply README.md:163]: Could you add the following envs as well?
- SYNC_MODE (full | fast)
- ENABLE_REST_FUNCTION_CALL (true | false)
- STAKE

File: start_node_docker.sh [hunk 4]
```diff
@@ -0,0 +1,57 @@
+if [[ $SEASON = 'mainnet' ]]; then
```
Comment (platfowner) [inline start_node_docker.sh:1]: question) where is $SEASON value originated in? Dockerfile or `docker run` command line?

Comment (kriii) [reply start_node_docker.sh:1]: Yes, I'll add info about it on README.

File: Dockerfile (hunk 1 above)
Comment (kriii) [reply Dockerfile]: Changed!

File: README.md (hunk 2 above)
Comment (kriii) [reply README.md]: I added this info (with SEASON), but It seems messy.
I'll describe each env individually a bit later.

Comment (kriii) [reply README.md]: Changed!

File: start_node_docker.sh [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+if [[ -z "$ACCOUNT_INJECTION_OPTION" ]]; then
+    printf 'You should manually inject your account into this node.\n'
+elif [[ $ACCOUNT_INJECTION_OPTION = "private_key" ]]; then
+    if [[ -z "$PRIVATE_KEY" ]]; then
+        printf "Must provide a PRIVATE_KEY\n"
+        exit
+    fi
+    echo $PRIVATE_KEY | node inject_account_gcp.js $NODE_ENDPOINT --private-key
+    unset PRIVATE_KEY
+elif [[ $ACCOUNT_INJECTION_OPTION = "keystore" ]]; then
```
Comment (platfowner) [inline start_node_docker.sh:36]: minor) what about adding a todo for "Support keystore file upload."?

File: README.md [hunk 6]
```diff
@@ -157,13 +156,33 @@ docker build -t ain-blockchain .
 ```
 - Pull Docker image
 ```
-docker pull ainblockchain/blockchain-database
+docker pull ainblockchain/ain-blockchain
 ```
-- Run with Docker image
+- Run with Docker image example
+```
+docker run -e SEASON=mainnet -e SYNC_MODE=fast -e ENABLE_REST_FUNCTION_CALL=true -e STAKE=10000 --network="host" -d ainblockchain/ain-blockchain:latest
```
Comment (platfowner) [inline README.md:163]: minor) what about providing sample command line with 'SEASON=dev'?

**Review by platfowner — COMMENTED** [review]
LGTM with some minor comments.
Thanks!
(Could you wait for liayoo's approval?)

File: README.md [hunk 7]
```diff
@@ -157,13 +156,33 @@ docker build -t ain-blockchain .
...
+```
+Each environment variables have the following options.
```
Comment (liayoo) [inline README.md:165]: 👍

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: start_node_docker.sh (hunk 5 above)
Comment (kriii) [reply start_node_docker.sh:36]: Added!

File: README.md (hunk 6 above)
Comment (kriii) [reply README.md:163]: Added!

**Comment by kriii** [comment]
Now merged! Thanks for the reviews!

**Pull request merged**